public/js/controllers/charts_controller.js

```diff
@@ -562,18 +591,19 @@ export default class extends Controller {
 
       case 'coin-supply': // supply graph
         d = circulationFunc(data)
+        this.customLimits = d.limits
         assign(gOptions, mapDygraphOptions(d.data, [xlabel, 'Coin Supply', 'Inflation Limit', 'Mix Rate'],
           true, 'Coin Supply (DCR)', true, false))
         gOptions.y2label = 'Inflation Limit'
         gOptions.y3label = 'Mix Rate'
         gOptions.series = { 'Inflation Limit': { axis: 'y2' }, 'Mix Rate': { axis: 'y3' } }
-        this.visibility = [true, true, this.anonymitySetTarget.checked]
+        this.visibility = [this.supplySetTarget.checked, this.predictedSetTarget.checked, this.anonymitySetTarget.checked, true]
```

Comment:
We have 3 lines. Why do you have four items here?

Reply:
Resolved.